Tutorial HTML/Html 01/README.md

```diff
@@ -0,0 +1,85 @@
+# Aplicatii web pentru incepatori
+######  Autor : Oancea Ionut Eugen
+
+![HACKED.io: Hands on FirefoxOS](https://arcturus.github.io/firefoxos-hackedio/img/html5_css_javascript.png)
+
+Orice aplicatie web acesibila intr-un browser foloseste o combinatie de 3 limbaje pentru a ajunge la forma sa finala (HTML + CSS + JavaScript). 
```

Comment:
> Orice aplicatie web accesibila intr-un browser

Nu e adevarat pentru toate aplicatiile Web? Chiar daca sunt intr-un WebView, tot cu HTML, CSS si JS ruleaza. As zice sa lasi doar `Orice aplicatie web foloseste...` Cu cat e mai putin text, si mai mai la obiet, cu atat mai bine, Focusul o sa fie pe exercitii oricum

Reply:
True, si da intradevar m-am gandit la asta.
Ca de exemplu poti faqce o pagina web doar in js si sa ai un html cu un div sau ceva de genul care sa acopere toata pagina.
Insa ca sa faci aplicatia sa apara pe tot ecranul, sau sa o pozitionezi pe mijlocul paginii ai nevoie de putin CSS
Am sa reformulez